### DISCUSSION TITLE
0.1.49

### DISCUSSION THREAD

**Discussion by eyberg** [nanovms/nanos#1989] [Announcements]
0.1.49

* preadv and pwritev support
* AMD SEV support
* additional options for getsockopt
* new support for extended reboot_on_exit options
* new PTP hardware clock
* support for TCP segmentation offload (TSO)

<hr /><em>This discussion was created from the release <a href='https://github.com/nanovms/nanos/releases/tag/0.1.49'>0.1.49</a>.</em>

**Comment by prologic** [comment]
Btw are there any release notes or a change log or what's new? 🤔

**Reply by eyberg** [reply]
Sure - I just added some salient ones. The actual list is much longer but can be obtained by looking at the diff from the tags.

**Reply by prologic** [reply]
Yeah I've watched the progress of this project for some years now and would love to see release notes 🙏

**Reply by eyberg** [reply]
fyi - just recognized your username - I think we have a few of your packages up on the repo: https://repo.ops.city/v2/packages/eyberg/yarn/0.15.1/x86_64/show (and more in our queue)

**Reply by prologic** [reply]
Oh wow! 😮
